src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1546,4 +1562,57 @@ public String toString()
                 + hashCode();
         }
     }
+
+    /**
+     * Extracts bosh connection sessionId, we just print it for debug
+     * purposes.
+     * @param connection the bosh connection which sessionId we will try to
+     * extract.
+     * @return the sessionId if extracted or null.
+     */
+    private static Object getConnSessionId(Object connection)
```

Comment:
It's not a big deal, but I would rather avoid putting utility methods in relatively large already `JvbConference` class.

Reply:
Changed.